### PR TITLE
web/portal: Added Special Number entity to portal platform

### PR DESCRIPTION
**Pull request by fmendozzac**
#### Type Of Change 
- [ ] Small bug fix
- [x] New feature or enhancement
- [ ] Breaking change

#### Checklist:
- [x] Commits are named and have tag following [commit rules](https://github.com/irontec/ivozprovider/blob/bleeding/doc/dev/en/commits.md)
- [x] Commits are split per component (schema, web/admin, kamusers, agis, ..)
- [x] Changes have been tested locally
- [ ] Fixes an existing issue (Fixes #XXXX) 
- [ ] Upport from existing Pull request #XXXX

#### Description
Added Special Number entity to portal platform

### CODE REVIEW

**Comment by ironArt3mis** [comment]
Rebase completed.

File: web/portal/platform/src/entities/Country/Country.tsx [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+  },
+};
+
+const country: EntityInterface = {
+  ...defaultEntityBehavior,
+  icon: SettingsApplications,
+  iden: 'Country',
+  title: _('Country', { count: 2 }),
+  path: '/countries',
+  toStr: (row: any) => {
```
Comment (mmadariaga) [inline web/portal/platform/src/entities/Country/Country.tsx:24]: any is forbidden

File: web/portal/platform/src/entities/Country/CountryNameSelectOptions.ts [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+  callback,
+  cancelToken,
+}): Promise<unknown> => {
+  const entities = store.getState().entities.entities;
+  const Country = entities.Country;
+
+  return defaultEntityBehavior.fetchFks(
+    Country.path,
+    ['id', 'name'],
+    (data: any) => {
```
Comment (mmadariaga) [inline web/portal/platform/src/entities/Country/CountryNameSelectOptions.ts:15]: any is forbidden

File: web/portal/platform/src/entities/Country/CountryNameSelectOptions.ts [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+      const options: any = {};
```
Comment (mmadariaga) [inline web/portal/platform/src/entities/Country/CountryNameSelectOptions.ts:16]: same here + use array options

File: web/portal/platform/src/entities/SpecialNumber/ForeignKeyGetter.tsx [hunk 4]
```diff
@@ -0,0 +1,22 @@
+import { ForeignKeyGetterType } from '@irontec/ivoz-ui/entities/EntityInterface';
+import { autoSelectOptions } from '@irontec/ivoz-ui/entities/DefaultEntityBehavior';
+import { SpecialNumberPropertyList } from './SpecialNumberProperties';
+
+/** TODO remove this file unless you need to change default behaviour **/
```
Comment (mmadariaga) [inline web/portal/platform/src/entities/SpecialNumber/ForeignKeyGetter.tsx:5]: remove this file

**Review by mmadariaga — CHANGES_REQUESTED** [review]

File: web/portal/platform/src/entities/Country/SelectOptions.ts [hunk 5]
```diff
@@ -0,0 +1,35 @@
...
+
+  return defaultEntityBehavior.fetchFks(
+    Country.path + `?_order[name.${[language]}]=ASC`,
+    ['id', 'name', 'countryCode'],
+    (data: Array<EntityValues>) => {
+      const options: DropdownChoices = [];
+      for (const item of data) {
+        options.push({
+          id: item.id as string,
+          label: `${(item.name as string)[language as any]} (${
```
Comment (mmadariaga) [inline web/portal/platform/src/entities/Country/SelectOptions.ts:23]: any is forbidden

**Review by mmadariaga — CHANGES_REQUESTED** [review]

**Pull request merged**